packages/ember/tests/routing/basic_test.js

```diff
@@ -2258,82 +2247,6 @@ test('The template is not re-rendered when the route\'s context changes', functi
   equal(insertionCount, 1, 'view should still have inserted only once');
 });
 
-
-test('The template is not re-rendered when two routes present the exact same template, view, & controller', function() {
```

Comment:
Not really.  It is confirming that `setOutletState` is stable if the controller/template/view are the same.  This test should be salvaged, and just remove usage of views from it (specify `templateName` in the routes instead).